src/Nethermind/Nethermind.Serialization.Rlp/AccountDecoder.cs

```diff
@@ -33,6 +33,18 @@ public AccountDecoder(bool slimFormat = false)
             return (codeHash, storageRoot);
         }
 
+        public (Hash256 CodeHash, Hash256 StorageRoot) DecodeHashesOnly(ref Rlp.ValueDecoderContext context)
```

Comment:
consider ValueHashes?

Reply:
Good point. I was porting to match the previous. Will look for the usages.

Reply:
Won't do. They are used in the `StateSyncItem`.